Bottom: d5040cc80a09258fc823c91e6b4780035fd4c067
Top:    ffa1c011087fcdc0f082e5a6c7f38dd34c12d6d8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index b9e5517..d77aecf 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -62,7 +62,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 48ddf29..7ac041c 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
